tests/clusterclient_async.c

```diff
@@ -65,22 +65,37 @@ void disconnectCallback(const redisAsyncContext *ac, int status) {
 }
 
 int main(int argc, char **argv) {
-    if (argc <= 1) {
-        fprintf(stderr, "Usage: clusterclient_async HOST:PORT\n");
+    int use_cluster_slots = 1; // Get topology via CLUSTER SLOTS
+
+    int optind;
+    for (optind = 1; optind < argc && argv[optind][0] == '-'; optind++) {
+        if (strcmp(argv[optind], "-n") == 0) {
+            use_cluster_slots = 0; // Get topology via CLUSTER NODES
+        } else {
+            fprintf(stderr, "Unknown argument: '%s'\n", argv[optind]);
+        }
+    }
+
+    if (optind >= argc) {
+        fprintf(stderr, "Usage: clusterclient_async [-n] HOST:PORT\n");
+        fprintf(stderr, "Options:\n");
+        fprintf(stderr, " -n  Get cluster topology using CLUSTER NODES\n");
```

Comment:
It's a slightly unintuitive syntax and default, considering hiredis-cluster uses CLUSTER NODES by default. But it's just test orchestration so I guess it doesn't matter.

Perhaps `--use-cluster-nodes` is more self-explanatory?

Reply:
Yes, I guess it would be better to have a flag that enabled the use of `cluster slots` in existing testcases, but that affects all existing testcases/binaries. Maybe we should change which is default in a coming release, but then there is `cluster shards` as well..

Updated to use `--use-cluster-nodes`

Reply:
Good. There is no reason to change all the test cases. It's just a test tool after all.